extensions/shared/paid-blocks/index.js

```diff
@@ -0,0 +1,65 @@
+/**
+ * External dependencies
+ */
+
+/**
+ * WordPress dependencies
+ */
+import { addFilter } from '@wordpress/hooks';
+import { Fragment } from '@wordpress/element';
+import { createHigherOrderComponent } from '@wordpress/compose';
+import { BlockControls } from '@wordpress/block-editor';
+import { ToolbarGroup, Button } from '@wordpress/components';
+import { useSelect } from '@wordpress/data';
+import { __ } from '@wordpress/i18n';
+
+/**
+ * Internal dependencies
+ */
+import { getUpgradeUrl, isUpgradable } from "./utils";
+
+const jetpackPaidBlockEdit = createHigherOrderComponent(
+	OriginalBlockEdit => props => {
+		const planSlug = 'value_bundle';
```

Comment:
Minor: it might be clearer to extract this to a constant and maybe even move it outside the function?

```suggestion
		const PLAN_SLUG = 'value_bundle';
```

Reply:
sorry, I pushed changes after seeing these suggestions. Updating locally.